feat(guide): port remaining prs/issues from legacy (#10974)
discordjs/discord.js: apps/guide/content/docs/voice/seeking.mdx

---
title: Seeking
---

> Discord.js does not have a built-in method for seeking audio resources. However external libraries such as `prism-media` can be used to tackle this issue.

Reviewer: ```suggestion
discord.js does not have a built-in method for seeking audio resources. However external libraries such as `prism-media` can be used to tackle this issue.
```

title: Seeking
---

> discord.js does not have a built-in method for seeking audio resources. However external libraries such as `prism-media` can be used to tackle this issue.

Reviewer 2: ```suggestion
There is no built-in method for seeking audio resources in discord.js. However external libraries such as `prism-media` can be used to tackle this issue.
```
Because I feel you on the grammar.
